src/test/java/com/wikia/webdriver/common/core/drivers/browsers/GhostBrowser.java

```diff
@@ -0,0 +1,32 @@
+package com.wikia.webdriver.common.core.drivers.browsers;
+
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.drivers.BrowserAbstract;
+
+import org.openqa.selenium.phantomjs.PhantomJSDriver;
+import org.openqa.selenium.phantomjs.PhantomJSDriverService;
+
+import java.io.File;
+
+public class GhostBrowser extends BrowserAbstract {
+
+  @Override
+  public  WikiaWebDriver setInstance(){
+    String phantomJSBinaryName;
+    String OSName = System.getProperty("os.name").toUpperCase();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00117)